codemods/components-lib-imports/mapping.ts

```diff
@@ -0,0 +1,73 @@
+export const mapping = {
+  // Basic remaps that should _just work_
+  Box: { name: 'Box', import: '@rbilabs/universal-components' },
+  VisuallyHidden: { name: 'VisuallyHidden', import: 'components/ucl' },
+  TextInput: { name: 'TextInput', import: 'components/ucl/text-input' },
+  ITextInputProps: { name: 'TextInputProps', import: 'components/ucl/text-input/types' },
+  Select: { name: 'Select', import: '@rbilabs/universal-components' },
+  Checkbox: { name: 'Checkbox', import: '@rbilabs/universal-components' },
+  useMediaQuery: { name: 'useMediaQuery', import: 'hooks/use-media-query', defaultImport: true },
+  NumberInput: { name: 'NumberInput', import: 'components/ucl/number-input' },
```

Comment:
This one is also in the index
```suggestion
  NumberInput: { name: 'NumberInput', import: 'components/ucl' },
```